packages/channel-client/src/channel-client.ts

```diff
@@ -5,10 +5,14 @@ import {
   ChannelResult,
   UnsubscribeFunction,
   Message,
-  Participant,
-  Allocation,
-  PushMessageResult
+  SiteBudget
 } from './types';
+import {
+  PushMessageResult,
+  Allocation,
+  Participant,
+  BudgetResult
+} from '@statechannels/client-api-schema/types';
```

Comment:
I think you could probably just import these from `@statechannels/client-api-schema`?